.github/workflows/workflow.yml

```diff
@@ -0,0 +1,29 @@
+# You can read more about github actions here: 
+# https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow
+# https://help.github.com/en/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
What is ubuntu-latest? the latest version ?

Reply:
Yeah, so it will just get whatever the latest version that currently exists is